stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml

```diff
@@ -324,7 +325,7 @@ spec:
         persistentVolumeClaim:
           claimName: {{ template "artifactory-ha.fullname" . }}-backup-pvc
       {{- end }}
-      {{- if .Values.artifactory.persistence.local }}
```

Comment:
FYI this change wasn't documented and isn't backward compatible and nearly affected one of our HA deployments

Reply:
@robbie-demuth - sorry about this. It was actually a fix as `persistence.local` was not documented and was not intended to be used...
Anything you think should be added here? You can open a new issue if needed.